tools/travis_mvn_watchdog.sh

```diff
@@ -164,7 +164,7 @@ watchdog () {
 
 # Check the final fat jar for illegal artifacts
 check_shaded_artifacts() {
-	jar tf build-target/lib/flink-dist-*.jar > allClasses
```

Comment:
I assume with the "-" this only matches something like `flink-dist-1.2-SNAPSHOT` or so and with the fix, it also matched `flink-dist.jar`...